package.json

```diff
@@ -83,6 +83,7 @@
     "@americanexpress/one-app-ducks": "^4.1.1",
     "@americanexpress/one-app-router": "^1.1.0",
     "@americanexpress/vitruvius": "^2.0.0",
+    "abort-controller": "^3.0.0",
```

Comment:
no package-lock.json change?

Reply:
its already provided by fetch-enhancers and was deduped so no change